### PR TITLE
Removed wildcard at end of @match

### PR DESCRIPTION
**Pull request by malachi26**
it made it kind of difficult to do a review when the delay was changed to 10-20 seconds, the page kept refreshing. so I removed the wildcard at the end of the @match urls

### CODE REVIEW

**Comment by Zomis** [comment]
There's a lot of changes that has been done here, while some of them could be useful, others are not something I want to merge into this.

There's no way I am going to include that "If you would like to donate, please send to..." for example.

File: moderator-tools/ReviewQueueNotification.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
 /** @preserve
 // ==UserScript==
 // @name Review Queue Notification
-// @author Malachi Edited Simon Forsberg Created
-// @description Shows a desktop notification when there review items in the queue.
-// @namespace https://github.com/Zomis/SE-Scripts
+// @author Malachi with help from Simon Forsberg
```
Comment (Phrancis) [inline moderator-tools/ReviewQueueNotification.js:4]: Might I suggest removing @author tags? See [this link](http://stackoverflow.com/a/17271433/3626537). If one is interested in authorship, all one has to do is check the VCS history.

 Also I think the proper place for a PayPal link would be in the README.md file in the repository, not in the code itself, as it doesn't say anything useful about the code.

**Comment by malachi26** [comment]
there were some things that I didn't want to merge with your code Zomis. I apologize, I don't have a good grasp on using github just yet. I agree with your conclusion.  I am guessing that github automatically grabbed some changes from after I submitted the merge request but before you guys checked it out?????

**Comment by skiwi2** [comment]
Correct, when you open a Pull Request, then you are requesting your branch to be merged in target branch. So all commits you make after opening the PR are "added".

FYI, in this image you can see when you synchronized your PR, that is, when you added new commits on an open PR. Unfortunately the actual opening of the PR is not in the list, probably Duga was down.

schermafdruk 2015-08-18 18 53 55

**Pull request closed without merging**